src/schemas/json/bxci.schema-2.x.json

```diff
@@ -151,9 +151,22 @@
         "publish": {
           "$ref": "#/definitions/outputDockerPublish",
           "description": "Docker publishing information. If not present, no images will be published"
+        },
+        "vulnerability_scan": {
+          "type": "object",
+          "description": "Configuration for running vulnerability scans on published Docker images",
```

Comment:
Optional fix: link to docs can be added at the end of description in the following format: `\n{{url}}`.

Reply:
Thank you for the info. Sadly I don't have public docs for this.